### DISCUSSION TITLE
installazione di hacs

### DISCUSSION THREAD

**Discussion by cledi55** [indomus/forum#2843] [HUB: Home Assistant]
salve, cerco un aiuto per installare hacs su home assistant installato sul synology nas. Ho letto la guida, ho seguito le istruzioni ma dopo caricato il file che ho scaricato da git hub, mi dice che manca il file o la directory. potreste aiutarmi ?

**Comment by snakuzzo** [comment]
con quello che hai detto, sicuramente non possiamo aiutarti. Devi dettagliare tutto...non hai nemmeno indicato che tipo di installazione hai (HA OS su VM o Core su Docker). Poi che vuol dire "dopo caricato il file" ?
Devi essere più preciso

**Reply by cledi55** [reply]
Allora, scusate per l'inesperienza. L'installazione è su doker, sono riuscito anche a caricare, nella sottocartella "config" il file Hacs che avevo preso dal sito git hub ma torno su home assistant e nulla cambia, anche dopo il riavvio. Sto saltando qualche passaggio ? Grazie e scusate ancora per la poca chiarezza.

**Reply by snakuzzo** [reply]
Sì...rileggi la guida passo passo

**Reply by snakuzzo** [reply]
In sostanza devi aprire una Shell del container, andare sotto `/config`, scaricare e lanciare lo script con il comando...

```
wget -O - https://get.hacs.xyz | bash -
```

Al termine, riavviare Core

Fine
